internal/benchmarks/src/config.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import { bench } from 'vitest';
+
+export const isDevnet = process.env.DEVNET_WALLET_PVT_KEY !== undefined;
+
+const iterations = isDevnet ? 1 : 10;
+
+export const runBenchmark = (name: string, benchmarkFn: () => Promise<void>) => {
+  bench(
+    isDevnet ? name : `${name} (x${iterations} times)`,
```

Comment:
Is this a big enough value add to justify having a function that we need to wrap every test?

We could just import the iterations value from a config file, that is used against the `describe`?

Reply:
This arose out of a refactor where we had the `isDevnet` conditional on each test + adding the iterations per test, **but** not all tests use the 10 iterations though. So this refactor is the sweet spot I feel.

Reply:
Just floating ideas, what about:

```ts
// config.ts
export const isDevnet = process.env.DEVNET_WALLET_PVT_KEY !== undefined;
export const iterations = isDevnet ? 1 : 10;
```
```ts
// bench.test.ts
import { iterations } from './config.ts';

describe('benchmarks', () => {
  bench('tests a thing', () => {});
  bench('tests another thing thing', () => {});
}, { iterations });
```

Then we don't need to augment `bench` and can use the other config options if we need them.

Reply:
I personally like having the iterations in the name, as it's explicit to why one bench takes for example: 100ms and another takes 1000ms.

Reply:
Based on our sync today I clarified this refactor with @danielbate 

Also to corroborate @petertonysmith94 it's actually **required** to have these distinctions in the test names so that we get accurate reporting on Codspeed, otherwise the statistics will be convoluted (single iterations mixed with multiple iterations)

Reply:
Resolving due to codspeed requirement, thankyou for the clarification.

Reply:
@danielbate can we resolve this?